openreview/journal/invitation.py

```diff
@@ -4278,7 +4368,15 @@ def set_comment_invitation(self):
             'writers': [venue_id],
             'signatures': [venue_id],
             'edit': {
-                'signatures': { 'param': { 'regex': f"{editors_in_chief_id}|{self.journal.get_action_editors_id(number='${5/content/noteNumber/value}')}|{self.journal.get_reviewers_id(number='${5/content/noteNumber/value}', anon=True)}.*|{self.journal.get_authors_id(number='${5/content/noteNumber/value}')}" }},
+                'signatures': { 
+                    'param': { 
+                        'items': [ 
+                            { 'value': editors_in_chief_id, 'optional': True },
+                            { 'value': self.journal.get_action_editors_id(number='${7/content/noteNumber/value}'), 'optional': True },
+                            { 'prefix': self.journal.get_reviewers_id(number='${7/content/noteNumber/value}', anon=True), 'optional': True }, 
```

Comment:
Should the reviewers anon id have '.*' at the end?

I do not understand why some appear as:
`{ 'prefix': f"{self.journal.get_reviewers_id(number='${7/content/noteNumber/value}', anon=True)}.*", 'optional': True },`
and some as:
`{ 'prefix': self.journal.get_reviewers_id(number='${7/content/noteNumber/value}', anon=True), 'optional': True }`

Reply:
Just FYI: it shouldn't matter. The API removes the `.*`, but I think it's cleaner if it doesn't show up.